oap-server/server-starter/src/main/resources/ui-initialized-templates/so11y_oap/so11y-instance.json

```diff
@@ -1,19 +1,3 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
```

Comment:
Why remove this? I think we have header in other JSON files too.

Reply:
> Why remove this? I think we have header in other JSON files too.

It was a mistake and I have reverted, but I doubt we should have headers for JSON files, for standard JSON specification comments are not allow, hence if we want to manipulate these JSON files, most of the tools doesn't support JSON comments and it'll be a trouble.

Reply:
I am fine we remove them, just remove all of them. Also, I remember there are codes to deal with this JSON header, you need to remove them together.